src/openforms/submissions/api/views.py

```diff
@@ -106,3 +109,32 @@ def get(self, request, *args, **kwargs):
     def perform_destroy(self, instance):
         remove_upload_from_session(instance, self.request.session)
         instance.delete()
+
+
+@extend_schema(
+    summary=_("View suspended submissions."),
+)
+class SuspendedSubmissionListView(ListAPIView):
+    serializer_class = SuspendedSubmissionSerializer
+    pagination_class = PageNumberPagination
+
+    queryset = Submission.objects.filter(
+        completed_on__isnull=True,
+        suspended_on__isnull=False,
+    )
+
+    # TODO authentication? permissions?
+    # TODO we need something here to verify current requests is allowed to access this BSN
```

Comment:
from the API spec I saw the external API is expected to use TokenAuthentication, so we could connect this with a (custom) user permission in the backend.

Since it's an API client and not a real user, afaik there's no way to limit which BSNs are valid. Note that the end-user needs to re-authenticate for the form anyway, so even with a mismatching BSN this will be caught during the auth fase.